internal/controllers/cryostat_controller_test.go

```diff
@@ -2669,7 +2673,9 @@ func (t *cryostatTestInput) checkReportsDeployment() {
 	}))
 	Expect(metav1.IsControlledBy(deployment, cr)).To(BeTrue())
 	Expect(deployment.Spec.Selector).To(Equal(test.NewReportsDeploymentSelector()))
+	Expect(deployment.Spec.Replicas).ToNot(BeNil())
 	Expect(*deployment.Spec.Replicas).To(Equal(t.reportReplicas))
+	Expect(deployment.Spec.Strategy).To(BeZero())
```

Comment:
Not sure if I asked somewhere already, but is this fakeClient limitations because I suppose it would default to ''rollingUpdate" on actual cluster?

Reply:
Right, if left unset on a real cluster, Kubernetes will populate the Strategy field as `RollingUpdate`: https://github.com/kubernetes/kubernetes/blob/520b991347b9c77635c0e4555f1703a86dcdd4ff/pkg/apis/apps/v1/defaults.go#L46-L64

Reply:
Right! Make sense! Thanks :D